django_modern_rest/controller.py

```diff
@@ -87,6 +88,11 @@ def __init_subclass__(cls) -> None:
         }
         cls._validate_endpoints()
 
+        cls._serializer_context = SerializerContext.build_for_class(
```

Comment:
Please, make sure that `SerializerContext` is easy to customize. Use similar pattern that we use for `Endpoint`